utils/contributors.js

```diff
@@ -59,6 +59,32 @@ const CONTRIBUTERS = [
         `,
         github: "https://github.com/debarati-06",
     },
+       {
+        handle_name: 'chiggshiggs',
+        name: "Chirag Goel",
+        image: "/images/MYphoto.png",
```

Comment:
Image type is incorrect. You have attached `.jpeg` file and used `.png` in path.